src/test/java/support/fakes/FakeOkapi.java

```diff
@@ -97,6 +98,8 @@ private void registerFakeItemsStorageModule(Router router) {
       .withRootPath("/item-storage/items")
       .withRequiredProperties("materialTypeId", "permanentLoanTypeId")
       .withDefault("status", new JsonObject().put("name", "Available"))
+      .withCreateProcessors(new ItemCreateUpdateEffectiveLocationProcessor())
```

Comment:
Why does this code need to support multiple processors when the only use case we have has a single processor?